src/DynamoCore/Nodes/NodeCategories.cs

```diff
@@ -411,6 +496,57 @@ internal static string MakeAbsolutePath(string basePath, string relativePath)
             return resultUri.LocalPath;
         }
 
+        /// <summary>
+        /// Add spaces to string before capital letters e.g. CoordinateSystem to Coordinate System.
+        /// </summary>
+        /// <param name="original">incoming string</param>
+        internal static string InsertSpacesToString(string original)
+        {
+            if (string.IsNullOrWhiteSpace(original))
+                return "";
+            StringBuilder newText = new StringBuilder(original.Length * 2);
+            newText.Append(original[0]);
+            for (int i = 1; i < original.Length; i++)
+            {
+                // We also have to check was previous character capital letter, e.g. Import From CSV
+                // In future there won't be "Method(parA : int)",
+                // TODO: this additional check Equals('(') will be removed.
```

Comment:
Is this `TODO` still applicable? If so please make sure there is a task written to follow up on this and insert the link here in comment.